turbo.json

```diff
@@ -11,14 +11,12 @@
       "dependsOn": ["antlr4#build", "antlr4-c3#build"]
     },
     "antlr4-c3#build": {
-      "dependsOn": ["antlr4#build"]
+      "dependsOn": ["antlr4#build"],
+      "outputs": ["dist/**"]
```

Comment:
I think we need this specified to cache dist, given we had to declare `antlr4-c3#build` explicitly